orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -67,7 +67,8 @@ def forward(self, *inputs, **kwargs):
         try:
             # Exporting module to ONNX for the first time
             build_gradient_graph = False
-            if self._skip_check.is_set(_SkipCheck.SKIP_CHECK_BUILD_GRADIENT) == False:
+            if self._skip_check.is_set(_SkipCheck.SKIP_CHECK_BUILD_GRADIENT) == False or \
```

Comment:
pls add comments - if it's enabled, we will execute the logic once for the first forward, and skip it for the rest of the training. same for the one below, thx

Reply:
Added a comment.